examples/README.md

```diff
@@ -0,0 +1,93 @@
+# STAC Examples
+
+This directory contains various examples for all parts of the STAC specification. It is structured to be two valid STACs, meaning both [catalog.json](catalog.json) and [collection.json](collection.json) should successfully load in various tools. They do not follow *all* the [best practices](../best-practices.md) for STAC, mostly
+due to the fact that they contrive examples to show the spec and we are hosting in GitHub. But we note below where they differ from an ideal catalog.
+
+The various fields are mostly fictional, to be able to demonstrate the various aspects of the spec as tersely as possible. To get a sense
+of real world STAC implementations we recommend exploring the [stac-examples](http://github.com/stac-utils/stac-examples) repo, which 
+gathers in one place copies of STAC [Items](../item-spec/item-spec.md) and [Collection](../collection-spec/collection-spec.md) 
+from a number of different production catalogs that all follow good STAC practices. And you should also explore the various catalogs
+listed on [STAC Index](http://stacindex.org), to see full catalogs in production.
+
+## Organization
+
+This directory contains two STAC implementations, both valid, but simplified a bit from following all the best practices. 
+
+### Simple Collection
+
+This STAC implementation consists of three files, all contained at the root of the examples directory
+
+**[collection.json](collection.json)** is a minimal Collection, that links to three items. 
+
+**[simple-item.json](simple-item.json)** is the most minimal possible compliant Item record. Most all data will
+include additional fields, as STAC is designed to be a minimal common subset. But it is useful for showing exactly what is
+required.
+
+**[full-item.json](full-item.json)** is a more realistic example, for a hypothetical analytic image 
```

Comment:
The name sounds a bit weird. Is full < extended? Full sounds "all in", but then below it is extended? If it's just core, maybe core-item?

Reply:
Yeah, that sounds good, will change.